src/main/java/org/example/jvspringbootfirstbook/JvSpringBootFirstBookApplication.java

```diff
@@ -1,22 +1,54 @@
 package org.example.jvspringbootfirstbook;
 
+import java.math.BigDecimal;
+import org.example.jvspringbootfirstbook.model.Book;
+import org.example.jvspringbootfirstbook.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class JvSpringBootFirstBookApplication {
+    private static final String AUTHOR_GEORGE_ORWELL = "George Orwell";
+    private static final BigDecimal PRICE_11_99 = BigDecimal.valueOf(11.99);
+
+    @Autowired
+    private BookService bookService;
 
     public static void main(String[] args) {
         SpringApplication.run(JvSpringBootFirstBookApplication.class, args);
     }
+
     @Bean
     public CommandLineRunner commandLineRunner() {
         return new CommandLineRunner() {
             public void run(String... args) throws Exception {
+                Book firstBook = bookCreation("1984", AUTHOR_GEORGE_ORWELL,
```

Comment:
The convention in Java is for the methods to be verbs, e.g createBook()
You can have a look here: https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names